plow/backends/js/js_webfont.flow

```diff
@@ -28,7 +28,7 @@ prepareWebFonts(cfg : JsConfig) {
 					v = getJsonFieldValue(jsn, e.first, JsonNull());
 					if (v != JsonNull()) {
 						fl_name = resourceprefix + e.first + resourcesuffix;
-						saveFileContent(fl_name, json2string(v));
+						setFileContent(fl_name, json2string(v));
```

Comment:
Add error check so we get an error if it fails. Also above

Reply:
'plSaveFileContent(filename : string, content : string) -> void' function is added to plow/backends/utils.